packages/chat-core/src/index.ts

```diff
@@ -586,6 +954,7 @@ export function promisify<T>(
   return async (payload: T) => {
     return await new Promise((resolve, reject) => {
       const timeoutId = setTimeout(() => {
+        // TODO should this unsubscribe?
```

Comment:
should this timeout also call `convo.unsubscribe(subscription);` ?

Reply:
Yes it should!